Tema 6/tema6.js

```diff
@@ -0,0 +1,49 @@
+// 1. Funcția care creează un obiect pentru rețeta preferată
+
+function createRecipe(titlu, portii, ingrediente) {
+    const reteta = {
+        titlu: titlu,
+        portii: portii,
+        ingrediente: ingrediente,
```

Comment:
```suggestion
        ingrediente: [...ingrediente],
```
ideal ar fi sa cream o lista noua, ca sa nu sufere modificari igredientele din aceasta reteta daca se modifica in afara